regridding/_regrid/_regrid_from_weights.py

```diff
@@ -66,13 +83,15 @@ def regrid_from_weights(
             ),
         )
         values_output = np.zeros_like(values_input, shape=shape_output)
+
```

Comment:
Remove extra line

Reply:
I can't tell what this is referring to.  Is it resolved?